.github/workflows/lint.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ 'ubuntu-24.04' ]
-        python-version: [ '3.9', '3.10', '3.11', '3.12' ]
+        python-version: [ '3.9', '3.10', '3.11', '3.12', '3.13' ]
```

Comment:
I was wondering if you could define a global constant for the different version so you don't have to change this for every single step. It seems like there is no _"good"_ solution only [this](https://github.com/orgs/community/discussions/46785) one. I'm not sure if I like it or not. I'll let you decide here.